content/en/blog/_posts/2022-08-14-pod-has-network-condition.md

```diff
@@ -0,0 +1,109 @@
+---
+layout: blog
+title: 'Kubernetes 1.25: PodHasNetwork condition for pods'
```

Comment:
Should this be `Kubernetes v1.25` instead?

Reply:
`Kubernetes v1.25` is accurate. The instance below in the text was incorrectly referring to `1.24` which is fixed.